### ISSUE TITLE
Warning messages when running with Elixir 1.9.1 & OTP 22

### ISSUE THREAD

**Issue by yudistrange** [jonasschmidt/ex_json_schema#49]
Running validation emits the following warning message:
```
warning: Map.take/2 with an Enumerable of keys that is not a list is deprecated.  Use a list of keys instead.
  (elixir) lib/map.ex:395: Map.take/2
  (ex_json_schema) lib/ex_json_schema/validator/properties.ex:12: ExJsonSchema.Validator.Properties.validate/3
```

This seems to be the offending line: 
https://github.com/jonasschmidt/ex_json_schema/blob/fd808676f1ecbd52d874fb3eca180bc476d20440/lib/ex_json_schema/validator/properties.ex#L73

**Comment by stephanos** [comment]
There are two PRs open to fix it. But it seems the maintainer has abandoned this project :(

**Comment by jonasschmidt** [comment]
Sorry for the long delay. I was on sabbatical and had my GitHub email still set to the company address, so I never got the notifications for the open issues. The PRs have been merged and released as version `0.6.2`.
